src/containers/Tenant/Tenant.tsx

```diff
@@ -95,12 +94,7 @@ export function Tenant(props: TenantProps) {
         currentData: currentItem,
         error,
         isLoading,
-    } = overviewApi.useGetOverviewQuery(
-        {path, database: tenantName},
-        {
-            pollingInterval: autoRefreshInterval,
-        },
-    );
+    } = overviewApi.useGetOverviewQuery({path, database: tenantName});
```

Comment:
We make this request to receive schema object types (they are not changed unless object is deleted and new object with the same name is created) and check access rights, so we don't need autorefresh here